tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script is responsible for remotely running tests on an ARM instance.
+# It should return a status code useful to the kokoro infrastructure.
+# It currently assumes an instance will be selected by the time this script begins running.
+
+if [ -z "$KOKORO_KEYSTORE_DIR" ]; then
+    echo "KOKORO_KEYSTORE_DIR is unset. This must be run from kokoro"
+    exit 1
+fi
+
+IDENTITY=${KOKORO_KEYSTORE_DIR}/73836_grpc_arm_instance_ssh_private_test_key1
+
+if [ -z "$INSTANCE" ]; then
+    echo "no INSTANCE set, aborting"
+    exit 1
+fi
+
+FILE=grpc_aws_experiment_remote.sh
```

Comment:
Sure. Though honestly I structured things this way to avoid the risk of typo , given limited static checking in bash.

How about REMOTE_SCRIPT as a compromise?

Reply:
Sg.